packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -126,6 +127,44 @@ export const DevServerMixin = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupExtensions(app, options) {
+        // TODO: support extensions options array syntax i.e. ['extension-a', {}]
```

Comment:
There is a util in the work that I did, that you might be able to take advantage of. I think it's called `expand` and it's in the `resolver-utils.js` file. 

Knowing that is your plan to address this before merging?

Reply:
i'd like to do that in another ticket, as I think I'm not 100% clear on the API and the requirements, so it's best if we can discuss it first and implement in a future PR.